pkg/scheduler/factory/factory_test.go

```diff
@@ -371,35 +335,57 @@ func TestBind(t *testing.T) {
 }
 
 func testBind(binding *v1.Binding, t *testing.T) {
-	handler := utiltesting.FakeHandler{
-		StatusCode:   200,
-		ResponseBody: "",
-		T:            t,
-	}
-	server := httptest.NewServer(&handler)
-	defer server.Close()
-	client := clientset.NewForConfigOrDie(&restclient.Config{Host: server.URL, ContentConfig: restclient.ContentConfig{GroupVersion: &schema.GroupVersion{Group: "", Version: "v1"}}})
+	testPod := &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{Name: "foo", Namespace: metav1.NamespaceDefault},
+		Spec:       apitesting.V1DeepEqualSafePodSpec(),
+	}
+	client := fake.NewSimpleClientset(&v1.PodList{Items: []v1.Pod{*testPod}})
+	expectedBody := runtime.EncodeOrDie(schedulertesting.Test.Codec(), binding)
+	requestReceived := false
+	client.PrependReactor("create", string(v1.ResourcePods), func(action clienttesting.Action) (bool, runtime.Object, error) {
```

Comment:
Do you know what the default reactor does with bindings? Does it do anything that can be observed by going in through the Clientset interface?

Reply:
@misterikkit The default reactor is to react to the action using a object tracker, and will store the object states. You're right, we could check the object using the Clientset interface here instead of the reactor.

Reply:
@misterikkit Updated, now it's simpler, PTAL.